pkgs/os-specific/linux/firmware/raspberrypi/default.nix

```diff
@@ -1,15 +1,21 @@
 { lib, stdenvNoCC, fetchFromGitHub }:
 
+let verinfo = {
+  # version = "2022-01-06";
```

Comment:
note: I took the easy way out here and changed the external package version. Not sure this was the right call.